addons/product/models/product_template.py

```diff
@@ -392,6 +392,11 @@ def _onchange_uom(self):
         if self.uom_id and self.uom_po_id and self.uom_id.category_id != self.uom_po_id.category_id:
             self.uom_po_id = self.uom_id
 
+    @api.onchange('type')
+    def _onchange_type(self):
+        # Do nothing but needed for inheritance
+        return {}
+
```

Comment:
Is this really necessary?

Reply:
I think that you should also make sure that stock calls super then, isn't it?

Reply:
yes, I changed it here: https://github.com/odoo/odoo/pull/93624/files#diff-1f24ce9f94c5795040749acca5924384d7d17c0ac39b1993cef3b484e4bd30afR827

Reply:
Oupsie ok I did not see it. Thanks 🙂